docs/en/ingest-management/elastic-agent/otel-agent.asciidoc

```diff
@@ -0,0 +1,145 @@
+[[otel-agent]]
+= Run {agent} as an OTel Collector
+
+preview::[]
+
+The link:https://opentelemetry.io/docs/collector/[OpenTelemetry Collector] is a vendor-neutral way to receive, process, and export telemetry data. {agent} includes an embedded OTel Collector, enabling you to instrument your applications and infrastructure once, and send data to multiple vendors and backends. 
+
+When you run {agent} in `otel` mode it supports the standard OTel Collector configuration format that defines a set of receivers, processors, exporters, and connectors. Logs, metrics, and traces can be ingested using OpenTelemetry data formats.
+
+The following steps will help you to get set up quickly using {agent} as an OTel Collector. Note that all steps are run with `sudo` as it's generally recommended to run {agent} with administrative privileges.
+
+[discrete]
+== Run {agent} in OTel mode
+
+For simplicity, these steps demonstrate OTel mode using standalone {agent}. The install steps for {fleet}-managed {agent} are very similar. Refer to the <<elastic-agent-installation,Install {agents}>> section for the full set of requirements and steps for any install scenario.
+
+. On your host, download and extract the {agent} installation package. 
++
+--
+// tag::install-elastic-agent[]
+
+include::{ingest-docs-root}/docs/en/ingest-management/tab-widgets/download-widget.asciidoc[]
+
+// end::install-elastic-agent[]
+--
++
+The commands shown are for AMD platforms, but ARM packages are also available.
```

Comment:
We have cross platform download/install commands available that we will include in our onboarding guide that we can also leverage here!

Reply:
That `include::` statement up above at line 26 will pull in the install instructions for all platforms. It should render like this:

![Screenshot 2024-06-18 at 12 02 03 PM](https://github.com/elastic/ingest-docs/assets/41695641/65f0b5ff-10d3-4b7a-abd1-c4f94ae4c6fd)